tests/test-tokenizer-random.py

```diff
@@ -324,8 +339,20 @@ def func_tokenize2(text: str):
     # import os
     # tokenizers = os.listdir(path_tokenizers)
     tokenizers = [
-        "llama-spm",   # SPM
-        "phi-3",       # SPM
+        # "llama-spm",   # SPM
+        # "phi-3",       # SPM
+        # "bert-bge",    # WPM
+        # "jina-v2-en",  # WPM
```

Comment:
Yeah, I don't think these are needed. Maybe for WPM, but WPM (BERT) defaults to BPE (GPT). Still don't know enough about it, yet.